src/core/components/model.jsx

```diff
@@ -3,6 +3,16 @@ import ImmutablePureComponent from "react-immutable-pure-component"
 import ImPropTypes from "react-immutable-proptypes"
 import PropTypes from "prop-types"
 
+let decodeRefName = uri => {
```

Comment:
```suggestion
const decodeRefName = uri => {
```

Reply:
We broke away from the `let` some time ago, and we use now industry strandard practices and use `const` whenever possible.